test/libsolidity/util/BytesUtils.cpp

```diff
@@ -327,7 +364,13 @@ string BytesUtils::formatBytesRange(
 	{
 		bytes byteRange{it, it + static_cast<long>(parameter.abiType.size)};
 
-		if (!parameter.matchesBytes(byteRange))
+		if (!parameter.matchesBytes(byteRange) ||
```

Comment:
coding style

Reply:
What is the coding style problem here? Looks like that I'm blind.